<launch>
  <node name="traj_opti_node1" pkg="grad_traj_optimization" type="compare2" output="screen">
    <param name="alg" value="24" type="int"/>
    <param name="time_limit_1" value="0.4" type="double"/>
    <param name="time_limit_2" value="0.01" type="double"/>
    <param name="dt" value="0.2" type="double"/>

    <param name="ws" value="20.0" type="double"/>
    <param name="wc" value="1.0" type="double"/>

    <param name="alpha" value="10.0" type="double"/>
    <param name="d0" value="0.8" type="double"/>
    <param name="r" value="0.5" type="double"/>

    <param name="alpha_v" value="0.0" type="double"/>
    <param name="v0" value="2.5" type="double"/>
    <param name="r_v" value="1.5" type="double"/>

    <param name="alpha_a" value="0.0" type="double"/>
    <param name="a0" value="3.5" type="double"/>
    <param name="r_a" value="1.5" type="double"/>

    <param name="bos" value="3.0" type="double"/>
    <param name="vos" value="8.0" type="double"/>
    <param name="aos" value="10.0" type="double"/>

    <param name="mean_v" value="1.8" type="double"/>
    <param name="init_time" value="0.3" type="double"/>

  </node>
</launch>
 
